client/src/components/Banner.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect } from 'react'
+import { useBanner } from 'hooks/useBanner'
+import { Box, Paragraph } from 'grommet'
+import { Link } from 'components/Link'
+import { Icon } from 'components/Icon'
+import styled, { css } from 'styled-components'
+
+export const Banner = ({
+  bannerLabel,
+  bgColor = 'brand',
+  ctaLabel,
+  ctaLink,
+  fontColor = 'white',
+  htmlIcon,
+  iconName,
+  iconSize,
+  children
+}) => {
+  const { showing, setShowing, onBannerClose } = useBanner()
+  const BannerBox = styled(Box)`
+    ${({ theme }) => css`
+      background-color: ${theme.global.colors[bgColor] ||
+      theme.global.colors.gradient[bgColor]};
+      > div {
```

Comment:
can you remove this style rule, this would be the responsibility of the person defining `children` to implement